Stripe/PublicHeaders/STPPaymentHandler.h

```diff
@@ -130,7 +130,7 @@ NS_EXTENSION_UNAVAILABLE("STPPaymentHandler is not available in extensions")
  
  Call this method if you are using automatic confirmation.  @see https://stripe.com/docs/payments/payment-intents/ios
  
- @param paymentParams The params used to confirm the PaymentIntent.
+ @param paymentParams The params used to confirm the PaymentIntent. Note that use of STPPaymentHandler enforces that a useStripeSDK value of true is set during the confirm call, regardless of paymentParams.useStripeSDK value.
```

Comment:
brevity nit, feel free to take or leave: "Note that this method overrides the value of paymentParams.useStripeSDK to `YES`".

Reply:
yeah I like that better. thanks!